test/cluster/test_tablets_migration.py

```
@@ -37,6 +37,12 @@ async def await_api_task(task, allowed_exception: Optional[Type[Exception]]=None
             raise
 
 
+async def get_cache_partitions(manager: ManagerClient, server, shard: int) -> float:
```

Ruff (0.16.3)

[error] Undefined name `ManagerClient`

(F821)